test/nuxeo-connection.html

```diff
@@ -99,8 +99,11 @@
           });
 
           test('should run the next', () => {
+            const connectedFired = new Promise((resolve) => {
+              document.addEventListener('connected', (e) => resolve(e));
+            });
             const connection = fixture('nxc-ok');
-            return timePasses(100).then(() => {
+            return connectedFired.then(() => {
```

Comment:
We are repeating this method three times, is it possible to refactor this into a separate one to avoid repetition?